chef/cookbooks/ceph/recipes/radosgw.rb

```diff
@@ -7,6 +7,9 @@
   include_recipe "ceph::keyring"
 end
 
+control_node = get_instance("roles:keystone-server")
+keystone_settings = KeystoneHelper.keystone_settings(control_node, @cookbook_name)
```

Comment:
Lint/UselessAssignment: Useless assignment to variable - `keystone_settings`. (https://github.com/bbatsov/ruby-style-guide#underscore-unused-vars)